src/idl_gen_python.cpp

```diff
@@ -683,6 +683,19 @@ class PythonGenerator : public BaseGenerator {
     return true;
   }
 
+  std::string genFileIdentifier(const std::string namespace_dir) {
+      std::string code = "";
+      BeginFile(namespace_dir, false, &code);
+      code += "def GetFileIdentifier():";
```

Comment:
It's ok to do multiple strings on one line, efficiency is not as big a deal here.

Reply:
done